app/views/fields/has_many/_form.html.erb

```diff
@@ -20,7 +20,7 @@ and is augmented with [Selectize].
 %>
 
 <div class="field-unit__label">
-  <%= f.label field.attribute_key, field.attribute %>
+  <%= f.label field.attribute, for: "#{f.object_name}_#{field.attribute_key}" %>
```

Comment:
If there is an existing Rails helper I can use to generate this label key, I think that would be nice, but I couldn't find one.